ospd/ospd.py

```diff
@@ -1330,7 +1332,8 @@ def get_vt_xml(self, vt_id: str):
         return vt_xml
 
     def get_vts_xml(self, vt_id: str = None, filtered_vts: Dict = None):
```

Comment:
I am a bit confused about this method now. Should it return a generator or a list? IMHO we should adjust it's name now too.